cro-clib/android.md

```diff
@@ -18,6 +18,15 @@ export NDK_HOME=$ANDROID_HOME/ndk/21.0.6113669
 export RUST_ANDROID=$HOME/rust_android/NDK
 ```
 
+## check crate-type
+1. vi Cargo.toml  in cro-clib folder
+2. modify crate-type as staticlib only  
+```
+[lib]
+name = "cro_clib"
+crate-type =["staticlib"]
+```
```

Comment:
Can this change be made directly in Cargo.toml? Most platforms need `staticlib`, so I guess it'll make sense to make it a default crate-type, so it's not necessary for someone to modify Cargo.toml (if someone really needs `cdylib` for some reason, they can tweak it... but for 99% cases, `staticlib` should be fine?)

Reply:
iOS also needs staticlib?

Reply:
yes, both staticlib
in ios, all libs are static

i'll modify

staticlib is find & stable